book/introduction.asc

```diff
@@ -1,6 +1,8 @@
 [preface]
 == Einleitung
 
+include::book/translation_note.asc[]
```

Comment:
Paths are relative, so here, use:
```
include::translation_notes.asc[]
```

Reply:
It works once, but now I get the same message again